Client-Side/shecodesprogramming/src/Components/NavBar/navbar.css

```diff
@@ -0,0 +1,108 @@
+.livebooks{
+    height: 55px;
+    width: 55px;
+    /* margin-top: 20px; */
+}
+
+.jtdlogo{
+    height: 50px;
+    width: 50px;
+}
+
+.tools{
+    margin-top: 400px;
+}
+
+.Account{
+    height: 55px;
+    width: 55px;
+}
+
+.main{
+    border: 3px solid black;
+    background-color: #D3D3D3;
+    width: 70px;
+    height: 800px;
+    border-top: none;
+    border-left: none;
+    border-bottom: none;
+    position: fixed;
+}
+
+
+ /* .livebooks {
+    color: white;
+    font-size: 5px;
+    border: none;
+    cursor: pointer;
```

Comment:
Please remove unwanted code.

Reply:
I removed unwanted code could you please review it.